server/sonar-server/src/main/java/org/sonar/server/organization/ws/EnableSupportAction.java

```diff
@@ -74,15 +81,21 @@ public void define(WebService.NewController context) {
 
   @Override
   public void handle(Request request, Response response) throws Exception {
+    verifySystemAdministrator();
+    
+    List<RuleKey> disabledTemplateAndCustomRuleKeys = null;
```

Comment:
no need to define this variable that early, move
`ruleIndexer.indexRuleDefinitions(disabledTemplateAndCustomRuleKeys);` after `dbSession.commit();` and get rid of the ugly variable initialized to `null`
closing the connection is not that important. The correct code would pass it to `ruleIndexer.indexRuleDefinitions` anyway (or it wouldn't really be closed so that a single SQL session is used by HTTP request)